persistence/mapper/src/main/scala/net/liftweb/mapper/MetaMapper.scala

```diff
@@ -1672,13 +1666,11 @@ object In {
     }
   }
 
-  def apply[InnerMapper <: Mapper[InnerMapper], JoinTypeA,
-            Zoom <% QueryParam[InnerMapper],
-            OuterMapper <: Mapper[OuterMapper]]
-  (_outerField: MappedField[JoinTypeA, OuterMapper],
-   _innerField: MappedField[JoinTypeA, InnerMapper],
-   qp: Zoom*): InThing[OuterMapper]
-  = {
+  def apply[InnerMapper <: Mapper[InnerMapper], JoinTypeA, Zoom, OuterMapper <: Mapper[OuterMapper]](
+    _outerField: MappedField[JoinTypeA, OuterMapper],
+    _innerField: MappedField[JoinTypeA, InnerMapper],
+    qp: Zoom*
+  )(implicit ev: Zoom => QueryParam[InnerMapper]): InThing[OuterMapper] = {
```

Comment:
Looks like we've also added implicits here that I'm not totally clear on the use of.

Reply:
As previously described, the deprecated view bound syntax was replaced by implicit function.

> * **View bounds deprecated**
>   * Deprecated: A <% B (#6500)
>   * Use instead: (implicit ev: A => B)

https://github.com/scala/scala/releases/tag/v2.13.0